modules/gapi/src/api/gproto.cpp

```diff
@@ -218,7 +219,9 @@ void cv::validate_input_arg(const GRunArg& arg)
     case GRunArg::index_of<cv::Mat>():
     {
         const auto desc = cv::descr_of(util::get<cv::Mat>(arg));
-        GAPI_Assert(desc.size.height != 0 && desc.size.width != 0 && "incorrect dimensions of Mat!"); break;
+        GAPI_Assert(desc.size.height != 0 && desc.size.width != 0 && "incorrect dimensions of Mat!");
+        GAPI_Assert(empty_gmat_desc() != desc && "empty cv::Mat is not allowed as runnable argument");
```

Comment:
Maybe it makes sense to replace `runnable` with `input`. We don't run Mats after all :)

Reply:
ok, it make sense.  would be replaced